sphinx_gallery/docs_resolv.py

```diff
@@ -336,11 +336,13 @@ def _embed_code_links(app, gallery_conf, gallery_dir):
                     src_gallery_dir,
                     relative=True)
             else:
-                doc_resolvers[this_module] = SphinxDocLinkResolver(url,
-                                                                   src_gallery_dir)
```

Comment:
Just a slightly awkward alignment which also made the line longer than 80 characters.